demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -811,6 +814,14 @@ def get_modified_and_added_files(self, compare_type, prev_ver):
         """
         if not self.no_configuration_prints:
             click.echo("Collecting all committed files")
+        if self.staged:
```

Comment:
when running in staged we want to run on all committed and staged files. If I get this correctly you are running on only staged files - unify with the results of the committed files section (actually a good opportunity to separate that section and this section to smaller commands to make the git handling more moduler.

Reply:
It's for the hooks - Need to run on the staged files only, not the committed. it's a pre-commit.